sklearn/metrics/_classification.py

```diff
@@ -2723,6 +2735,12 @@ def log_loss(
 
     # Renormalize
     y_pred_sum = y_pred.sum(axis=1)
+    if not np.isclose(y_pred_sum, 1, rtol=1e-15, atol=5 * eps).all():
```

Comment:
Form y education: why is it necessary to set the `rtol` and `atol` parameters ourselves instead of using the default values? The docs of `isclose` says that you should think about the values, but also that the default of `atol` is suitable for comparing values of order one.

Reply:
Ah, yes that makes sense. Thanks!